src/accordion/accordion-group.component.ts

```diff
@@ -76,9 +76,14 @@ export class AccordionPanelComponent implements OnInit, OnDestroy {
     this.accordion.removeGroup(this);
   }
 
-  toggleOpen(): void {
+  toggleOpen(event: Event): void {
+    if (this.checkAccordionPanelToggle(event.target as HTMLElement)) return;
     if (!this.isDisabled) {
       this.isOpen = !this.isOpen;
     }
   }
+
+  checkAccordionPanelToggle(target: HTMLElement): boolean {
```

Comment:
This is static method, don't add it to class and enable 
Or make it static or extract it to helper file